carl/envs/rna/data/parse_dot_brackets.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+
+
+def parse_dot_brackets(
+    dataset, data_dir, target_structure_ids=None, target_structure_path=None
+):
+    """TODO
```

Comment:
What TODO?

Reply:
That TODO was in the original, no clue what it refers to. As the functions does what it says, let's remove it?

Reply:
Yep

Reply:
Done